Adding the Juggler Archetype

spthiessen opened the pull request:
Adding the Bard archetype, Juggler.

marcus-downing commented:
Getting closer.

 - For consistency, the title should be in capital letters.
 - For clarity, the titles of abilities like "Deflect Arrows" should be in bold.
 - Even though it's a feat, you can drop the word "Gains", just use the ability name.
 - When an ability gets an incremental upgrade at subsequent levels, those should be grouped. In this case, the three entries for Deflect Arrows should be brought together, without hard lines in between.
 - When a number increases regularly (eg every four levels), it's best to put that into a calculation if you can; if not because the levels are awkward, group them into a table. So under combat juggling, you can start with a full explanation "You can hold or wield up to three weapons in your hands", and follow it up with a series of increments, "Up to four weapons", "Up to five weapons".
 - The white spaces under spells and Bardic Performance need redistributing between the other elements.

Now, I realise that my own work isn't 100% consistent. It's something that's evolved over time, and each class presents unique layout challenges.

marcus-downing commented:
([A link to the notes I left on the assets pull request](https://github.com/dyslexic-charactersheets/assets/pull/1))

marcus-downing commented:
I hope you see all these as suggestions to improve, not negative nit-picks. I don't want to discourage you at all.

marcus-downing commented:
You appear to have lost the Arcane Spell Failure Threshold?

spthiessen commented:
Thanks for the tips.

I think I have addressed your concerns with the latest version. I don't like the text, "Items Juggled" but can't think of a better one right now.

The missing "Spell Failure Threshold" was a mistake, it is in again.

I welcome your comments, I am playing this class right now and making it better now means it will be easier on me when I switch to this sheet, either at next level or the one after that (I printed a base bard before and crossed stuff out.)

marcus-downing commented:
Closer.

I think the calculation under Combat Juggling is probably too awkward after all, so it should probably be a table. Look at the Famous table under the Celebrity Bard for an example.

marcus-downing commented:
Apparently I've used 100% of my GitHub LFS quota for this month  😄  Might need to wait a bit before doing more things... or just pay them $5.

spthiessen commented:
Is that because of me?

On Thu, Feb 2, 2017 at 12:46 PM, Marcus Downing <notifications@github.com>
wrote:

> Apparently I've used 100% of my GitHub LFS quota for this month 😄 Might
> need to wait a bit before doing more things... or just pay them $5.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/dyslexic-charactersheets/pages/pull/1#issuecomment-277045509>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AYO3eY9P7XP8lPoK0OxIjcKqb3ERYGJEks5rYiSFgaJpZM4LyJ1F>
> .
>



-- 
Steve .T

marcus-downing commented:
I'm not sure how they calculate it. Maybe.

Anyway, the project gets enough donations on Patreon that paying $5 for a service I actually use doesn't bother me, so don't worry about it.

marcus-downing commented:
I just merged this, and noticed something odd: on importing into my version of Illustrator, it give a warning about importing from a newer version, then it split all of the text boxes up, on either newlines or font changes. This is a bad thing because it prevents the automatic translation script from working. The translator is looking for eg "Bard Level", but instead it sees "Bard" and then separately "Level".

@spthiessen , Can I ask you to save a copy of the file for CS6?

spthiessen commented:
Will do as soon as I can.

On Mar 11, 2017 3:19 PM, "Marcus Downing" <notifications@github.com> wrote:

I just merged this, and noticed something odd: on importing into my version
of Illustrator, it give a warning about importing from a newer version,
then it split all of the text boxes up, on either newlines or font changes.
This is a bad thing because it prevents the automatic translation script
from working. The translator is looking for eg "Bard Level", but instead it
sees "Bard" and then separately "Level".

@spthiessen <https://github.com/spthiessen> , Can I ask you to save a copy
of the file for CS6?

—
You are receiving this because you were mentioned.

Reply to this email directly, view it on GitHub
<https://github.com/dyslexic-charactersheets/pages/pull/1#issuecomment-285901031>,
or mute the thread
<https://github.com/notifications/unsubscribe-auth/AYO3eXXBvUEtItIKI0w093P2x9e2FQDuks5rkw_UgaJpZM4LyJ1F>
.

spthiessen commented:
OK saved as CS6 instead of CC.

Thank you.

marcus-downing commented:
Er... can you push the file?

marcus-downing commented:
Ah, you'd pushed to your own repo, but the pull request didn't pick up on that. I've grabbed the file from your own fork, and it looks better. Thanks!

(GitHub doesn't quite behave the same way as GitLab, which is what I'm used to using.)

spthiessen commented:
I tried pushing to your repo accidently at first. I dont have push rights,
so I put it in my repo. I think I'm going to delete my repo until I do
another archetype.

marcus-downing commented:
I've got your file merged and pushed now, so that's okay.

What are you thinking of doing next, now you know how the system works? I have a list of what people asked for. It's a very, very long list.

spthiessen commented:
I had none specifically chosen. Which is the most requested.

marcus-downing commented:
Fighter archetypes are the most requested, by far.

spthiessen commented:
OK, I'll start work on the Viking

marcus-downing commented:
Be warned, I'm in the middle of changing ALL the fonts. It may be best to wait a few days then pull the latest copy of the pages repo before you start.

spthiessen commented:
OK, do you think will I need to do a Melee and Ranged page?

marcus-downing commented:
It depends on the archetype - some of them are ranged-specific or melee-specific. But in many cases, yes, you'll need both. Look at [the axes and variants for the Druid class in the JSON](https://github.com/dyslexic-charactersheets/assets/blob/master/data/pathfinder.json#L741).

btw, chatting on a closed issue is kind of awkward, and GitHub doesn't have a PM feature. Can you send me a message on the website identifying yourself?

marcus-downing commented:
Hmmm. Come to think of it, the two fighter class sheets divide quite evenly. The left half is specific to the class/archetype, and the right half to the melee/ranged style. This suggests that with a little code and a touch of careful planning, we could save a lot of work.

marcus-downing commented:
All the pages with new fonts have been pushed to git. You'll need to install the new *Merriweather* and *Pangolin* fonts from the `pages/Fonts` folder.

This was prompted by Google deciding to change all the weights in the Merriweather font, meaning that the old Merriweather Black is closest to the new Merriweather Bold. I needed the new version to work with the Russian translation.

spthiessen commented:
Just sent the message on the website.
-- 
Steve .T

spthiessen commented:
Which is easier, me creating my own fork or creating branches on the main project? I think just creating a branch would be easier, but I don't have access. I can do my own fork though if that is easier.
